Sachhidanand/bit_array/definition.c

```diff
@@ -0,0 +1,18 @@
+//Definition of functions
+
+void b_create(int size_t){
```

Comment:
size_t is the type and not the variable name,
```suggestion
void b_create(size_t size){
```

Reply:
okay sir